app/models/slack_api_response.rb

```diff
@@ -0,0 +1,10 @@
+class SlackApiResponse < ActiveRecord::Base
+  
+  # Returns latest valid API response.
+  #
+  # @return [SlackApiResponse]
+  def self.latest(api_name)
+    self.where(method_name: api_name, ok: true).order(updated_at: :desc).first
```

Comment:
maybe add a limit(1) as well here so it doesnt waste time parsing all those records?